<!DOCTYPE html>
<html>
    {% include head.html %}
    <body>
        <div class="container">
          <div class="site">
            {% include header.html %}
            <div class="content">
              <div class="banner">
                <div class="photo">
                  <img src="/assets/me.jpg" width="140px" height="196px">
                </div>
                <div class="contact">
                  <a href="https://comp-path.bwh.harvard.edu/">Division of Computational Pathology</a>, Brigham and Women's Hospital<br>
                  Harvard Medical School<br>
                  <!--Hanes Hall B30<br> -->
                  <br>
                  Email: <a href="mailto:icarmichael@bwh.harvard.edu">icarmichael@bwh.harvard.edu</a> <br>
                  GitHub: <a href="https://github.com/idc9">idc9</a> <br>
                  <a href="https://www.linkedin.com/in/iain-carmichael-aa254183/">LinkedIn</a> <br>
                  <a href="https://scholar.google.com/citations?hl=en&user=7kfVUFQAAAAJ">Google Scholar</a> <br>
                </div>
              </div>
              <br>
              <br>
              <div class="homecontent">
                <b>Brief Bio</b> <br>

                I am a postdoc in <a href="https://faisal.ai/">Computational Pathology</a> at Harvard Medical School. Previously I was a postdoctoral fellow in the Department of Statistics at the University of Washington supported by the <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=1902440&HistoricalAwards=false">NSF Mathematical Sciences Postdoctoral Research Fellowship</a>. I received a PhD in statistics from the University of North Carolina at Chapel Hill where I was advised by <a href="https://shankarbhamidi.web.unc.edu/">Shankar Bhamidi</a> and <a href="https://marron.web.unc.edu/">J.S Marron</a>.

                <br><br>
                My research aims to solve challenging computational problems in biomedical sciences such as the analysis of very large scale medical images like <a href="https://www.pixelscientia.com/article-finding-prognostic-patterns-in-gigapixel-images.html">stained whole slide images</a> from Pathology. I’m broadly interested in developing statistical machine learning algorithms for data with a complex structure such as networks, images, and multi-view/modal data. The computational arm of my work weaves together statistics/probability, optimization, deep learning, and software engineering. The scientific arm of my work involves intensive collaboration with domain scientists/doctors and investigates basic scientific questions as well as clinical translational problems.

                <h3>Computational pathology and precision medicine</h3>

                Pathology lies at the heart of precision medicine; it is like the intellgience agency of medicine due to its central role in data collection and clinical decision making.
                Histological examination of a tissue biopsy has been the gold standard for cancer--and other disease--diagnosis for decades.
                <!---Pathology lives at the center of precision medicine and is like the intellgience agency of medicine due to its role in data collection and clinical decision making.--->
                <!---Pathology — sometimes described as the intelligence agency of medicine — lives at the center of precision medicine.--->
                New clinical measurement technologies (slide digitization, multiplex immunoflorecence, 3D histology, next-generation genomic sequencing, spatially resolved transcriptomics, microbiota characterization, etc) make it an exciting time for computational people to both impact clinical practice and help answer fundamental biology questions. My work in Pathology has two broad aims:
                <ul>
                <li>Use machine learning to augment clinical decision making for disease detection, diagnosis, prognosis, and response to treatment prediction.</li>

                <li>Use computational/data driven approaches to discover new disease biomarkers and subtypes.</li>
                </ul>

                From a computational/statistical perspective, Pathology is a gold mine for interesting problems; you can find almost any data type e.g. networks, images, shapes, point clouds, etc. Almost everything is at play; there are issues of robustness, fairness, computation (datasets can easily be multiple terabytes), deep learning interpretability, high-dimensional inference, etc. Two of the most important technical challenges in computational pathology include:
                <ul>
                <li>Deep learning with massive and complex images; whole slide images can be 100,000 x 100,000 pixels--over 100,000 times more pixels than a typical ImageNet image--and the visual signals tend to be complex, subtle, and heterogenous.</li>

                <li>Data integration for multiple data modalities such as images, molecular measurements, pathology reports, and clinical data.</li>
                </ul>


                <h3>Representative work</h3>

                <ul>
                <li> Carmichael, I., Song, A.H., Chen, R.J., Williamson, D.F.K., Chen, T.Y., Mahmood, F. (2022). Incorporating intratumoral heterogeneity into weakly-supervised deep learning models via variance pooling. <i> The International Conference on Medical Image Computing and Computer Assisted Intervention. To appear.</i> </li><br>

                <li> Carmichael, I. (2021). <a href="https://arxiv.org/pdf/2107.03494.pdf">The folded concave Laplacian spectral penalty learns block diagonal sparsity patterns with the strong oracle property</a>. <i> Under review.</i> </li><br>

                <li> Carmichael, I., Keefe, T., Giertych, N., Williams, J.P. (2021). <a href="https://arxiv.org/abs/2110.05567">yaglm: a Python package for fitting and tuning generalized linear models that supports structured, adaptive and non-convex penalties.</li><br>

                <li> Carmichael, I., Calhoun, B.C.,  Hoadley, K.A., Troester, M.A.,  Geradts, J., Couture, H.D.,  Olsson, L,.  Perou, C.M.,  Niethammer, M.,  Hannig, J., Marron, J.S. (2021). <a href="https://projecteuclid.org/journals/annals-of-applied-statistics/volume-15/issue-4/Joint-and-individual-analysis-of-breast-cancer-histologic-images-and/10.1214/20-AOAS1433.short">Joint and individual analysis of breast cancer histologic images and genomic covariates</a>. <i>The Annals of Applied Statistics, 15(4), pp.1697-1722.</i></li><br>

                <li> Carmichael, I. (2020). <a href="https://arxiv.org/pdf/2012.15313.pdf">Learning sparsity and block diagonal structure in multi-view mixture models</a>. <i> Under review.</i></li><br>


                <li> Banerjee, S., Bhamidi, S., Carmichael, I. (2018). <a href="https://arxiv.org/abs/1808.02439">Fluctuation bounds for continuous time branching processes and nonparametric change point detection in growing networks.</a> <i> Under review.</i></li>
                </ul>

                <br><br>

              </div>
            </div>
            <div class="sidebar"></div>
            {% include footer.html %}
          </div>
        </div>
      </div>
    </body>
</html>
